### PR TITLE
docs: Document new `prisma init` CLI parameters

### LINKED ISSUE #5359: Document new `prisma init` CLI parameters
Location https://www.prisma.io/docs/reference/api-reference/command-reference#init

See `--generator-provider`, `--preview-features` and `--output` from https://github.com/prisma/prisma/pull/21314

### PR DESCRIPTION
**Pull request by ruheni**
Closes #5359

### CODE REVIEW

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 1]
```diff
@@ -994,7 +997,7 @@ Other options:
 - Take the SQL script from standard input and execute it on the database specified by the data source URL given in the `DATABASE_URL` environment variable:
 
   ```terminal wrap
-  $ echo 'TRUNCATE TABLE dev;' | prisma db execute --stdin --url="$DATABASE_URL"
+  echo 'TRUNCATE TABLE dev;' | prisma db execute --stdin --url="$DATABASE_URL"
```
Comment (ruheni) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
  $ echo 'TRUNCATE TABLE dev;' | prisma db execute --stdin --url="$DATABASE_URL"
```

**Review by SevInf — APPROVED** [review]

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 2]
```diff
@@ -306,10 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| ----------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| `--datasource-provider` | No       | Specifies the default value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                         | `postgresql`                  |
+| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                |                               |
+| `--generator-provider`  | No       | Define the default Prisma Client generator provider to use.                                                                                                                                    | `prisma-client-js`            |
```
Comment (Jolg42) [inline content/400-reference/200-api-reference/200-command-reference.mdx]:
```suggestion
| `--generator-provider`  | No       | Define the default generator provider to use.                                                                                                                                    | `prisma-client-js`            |
```
Which can be Prisma Client or anything else.

**Review by Jolg42 — APPROVED** [review]
See comment

File: content/400-reference/200-api-reference/200-command-reference.mdx (hunk 2 above)
Comment (ruheni) [reply content/400-reference/200-api-reference/200-command-reference.mdx]: Thanks for the clarification!

**Review by Jolg42 — APPROVED** [review]

File: content/400-reference/200-api-reference/200-command-reference.mdx [hunk 3]
```diff
@@ -306,10 +306,13 @@ The `init` command does not interpret any existing files. Instead, it creates a
...
+| `--datasource-provider` | No       | Specifies the default value for the `provider` field in the `datasource` block. Options are `sqlite`, `postgresql`, `mysql`, `sqlserver`, `mongodb` and `cockroachdb`.                         | `postgresql`                  |
+| `--url`                 | No       | Define a custom datasource url.                                                                                                                                                                |                               |
+| `--generator-provider`  | No       | Define the default generator provider to use.                                                                                                                                                  | `prisma-client-js`            |
+| `--preview-features`    | No       | Define the default Preview features to use. [Learn more](/concepts/components/preview-features)                                                                                                |                               |
```
Comment (janpio) [inline content/400-reference/200-api-reference/200-command-reference.mdx:314]: Is this the only one that can be used multiple times to define the array? If so, we should mention that somehow as it is different from the other arguments that are singular.

Comment (ruheni) [reply content/400-reference/200-api-reference/200-command-reference.mdx:314]: Yes. I've created a small PR to take care of that with an example: https://github.com/prisma/docs/pull/5403

**Pull request merged**